site/content/en/docs/Installation/Terraform/eks.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Installing Agones on AWS Elastic Kubernetes Service using Terraform"
+linkTitle: "AWS"
+weight: 20
+description: >
+  You can use Terraform to provision an EKS cluster and install Agones on it.
+---
+
+{{% feature expiryVersion="1.3.0" %}}
```

Comment:
Rather than using `feature` shortcodes, you can set the `publishDate` at the top of the page to be the day of stable release:
https://agones.dev/site/docs/contribute/#at-the-page-level
https://gohugo.io/content-management/front-matter/